### ISSUE TITLE
Compliance Test support for SCIM 2.0

### ISSUE THREAD

**Issue by pvarga88** [erdtman/simplecloud.info#2]
This is just a formal request for SCIM 2.0 tests. I see the note 'Work on SCIM 2.0 tests is under development and there are currently no support for the enterprise extension'

Is the test suite under active development, or a nice-to-have in the unknown future?
Thanks

**Comment by erdtman** [comment]
There is no active development of the Compliance Tests.
If you would like to contribute I´m open to merge pull requests.

**Comment by damianh** [comment]
I see there has been some work done in this area:

- https://medium.com/@vindulajayawardana/scim-2-0-compliance-test-suite-737fd4ace3cc
- https://github.com/wso2-incubator/scim2-compliance-test-suite

But appears that is was a summer-of-code project and is not maintained (briefly judging from [the open issues](https://github.com/wso2-incubator/scim2-compliance-test-suite/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc)).

What are the plans here, if any?

**Comment by erdtman** [comment]
Yes, I have seen that and I asked the author to submit a PR to get it integrated with the site (should not be that hard since it was started from the same code base). However I have not heard anything so there is no active plans to integrate it.
